KeyShot/6.3.0/member_groups/metadata/metadata_editor.dcl

```diff
@@ -2,4 +2,8 @@ data member_group metadata_editor {
     name = 'Metadata editor'
     folder_id = data.member_group_folder.metadata.id
     ad_group_name = ''
+}
+
+patch member_group id_123 {
+    target = data.member_group.metadata_editor
```

Comment:
We need to ensure that we delete all empty patches - i.e. patches that don't have any other properties than `target`. Otherwise, the layer can not be imported in Configuration Management 😉